README.rst

```diff
@@ -186,7 +186,7 @@ http://loklak.org/api/search.rss?q=0rb1t3r or just follow @0rb1t3r
 Where can I find documentation?
 -------------------------------
 
-The Doumentation can be found at https://github.com/fossasia/susi_server/tree/development/docs/api/api.md
+The Documentation can be found at https://github.com/fossasia/susi_server/tree/development/docs/api/api.md
```

Comment:
Make it can be found [here](https://github.com/fossasia/susi_server/tree/development/docs/api/api.md)

Reply:
I have done that now and thanks for bringing that into notice